src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,51 @@
+.calendar {
+  $cell-size: 100px;
+  $cell-gap: 1px;
+  $days:
+    "mon" 0,
+    "tue" 1,
+    "wed" 2,
+    "thu" 3,
+    "fri" 4,
+    "sat" 5,
+    "sun" 6;
```

Comment:
usually variables are moved to a separate file

Reply:
I thought about this, but decided that these variables are used only in calendar file.